### DISCUSSION TITLE
What does it mean to visualize a TD?

### DISCUSSION THREAD

**Discussion by egekorkan** [thingweb/thingweb-playground#362] [Ideas]
@fatadel has implemented some initial ideas to show what it can mean to visualize TDs, which are available under the visualize button.

What other ideas does the community have? We are more than happy to support new implementations or implement new ideas and see which ones become more popular. 

The answer to "but why?" is also open. For me, it is a means to show and share TDs in a format that is not just JSON and maybe with enough TDs we can start to see some patterns 👀

**Comment by egekorkan** [comment]
![visu](https://user-images.githubusercontent.com/20195407/198335718-45fabf9c-c6a2-41b6-af1f-f1c951f583e2.gif)

**Comment by benfrancis** [comment]
An example from [WebThings Gateway](https://webthings.io/gateway/).

![Screen Shot 2022-10-27 at 16 51 48](https://user-images.githubusercontent.com/552417/198339753-8f1764ce-47b6-48b4-8f46-4c3af434f69d.png)
